python/scripts/report_on_repository_standards.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import logging
+
+from python.services.github_service import GithubService
+from python.services.operations_engineering_reports import OperationsEngineeringReportsService
+from python.services.standards_service import RepositoryReport
+
+
+def add_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--oauth-token",
+        type=str,
+        required=True,
+        help="The GitHub OAuth token to use",
+    )
+
+    parser.add_argument(
+        "--org",
+        type=str,
+        default="ministryofjustice",
+        help="The GitHub organisation to use",
+    )
+
+    parser.add_argument(
+        "--url",
+        type=str,
+        required=True,
+        help="The operations-engineering-reports url to use with the leading https:// and without the trailing /",
+    )
+
+    parser.add_argument(
```

Comment:
I believe the endpoint would be fine to be hard wired into the service layer, it's more of a technical detail of how we get the data we want

Unless we're expecting to need to change it at runtime? 🤔

Reply:
I believe there will be multiple hostnames depending on where it is running, i.e. local, dev, staging or prod. I would like to keep this as an argument.

Reply:
hostnames I agree with 😁  they don't change often, but you often have different environments as you say

Was referring to the endpoint that we call, more of a client implementation detail than something we should need configurable 🔧